moto/secretsmanager/models.py

```diff
@@ -565,6 +565,19 @@ def restore_secret(self, secret_id):
 
         return secret.arn, secret.name
 
+    def update_secret_tags(self, secret_id, tags):
```

Comment:
Let's keep the naming consistency here as in `responses.py`, so call this method `tag_resource` as well. 
This is done by all methods, to make it clear it implements that specific SDK call.